requirements.txt

```diff
@@ -62,7 +62,10 @@ Flask-RESTful==0.3.7
 Flask==1.1.1
 funcparserlib==0.3.6
 funcsigs==1.0.2
+#functools32 #==3.2.3.post2
 furl==2.0.0
+future==0.17.1
```

Comment:
I don't believe this is needed at all w/ Python3 since it's literally backported functionality from Python3. We can keep it in there to be safe, if you like though. Although I thought it was not installable on Python3.